.pre-commit-config.yaml

```diff
@@ -15,12 +20,16 @@ repos:
     hooks:
     -   id: codespell
         args: [--ignore-words=.dictionary.txt]
--   repo: https://github.com/pre-commit/mirrors-isort
-    rev: v5.10.1
+        additional_dependencies:
+        - tomli
+-   repo: https://github.com/pycqa/isort
+    rev: 5.11.5
     hooks:
-    -   id: isort # sort imports alphabetically and separates import into sections
-        args: [-w=88, -m=3, -tc, -sp=setup.cfg ]
+      - id: isort
+        name: isort (python)
+        args: [-w=88, -m=3, -tc]
```

Comment:
I am not completely sure, but I believe `isort` can read the config directly from pyproject.toml, also from pre-commit

Reply:
You're right, I've updated that. I've also switched to using the `profile` option which includes a set of defaults for `black` which match what we had: https://pycqa.github.io/isort/docs/configuration/profiles.html